db/migrate/20150709221755_create_logins.rb

```diff
@@ -1,14 +1,17 @@
 class CreateLogins < ActiveRecord::Migration
 
   def change
-    create_table :logins do |t|
+    primary_key_chooser = lambda do |key|
+      RailsApiAuth.primary_key_type ? { key => RailsApiAuth.primary_key_type } : {}
+    end
```

Comment:
would be cool to have this in a private method, e.g. `def key_option(option_name)`